src/snowflake/snowpark/modin/plugin/_internal/telemetry.py

```diff
@@ -571,9 +567,7 @@ def __new__(
             attrs (Dict[str, Any]): The attributes of the class.
 
         Returns:
-            Union[snowflake.snowpark.modin.pandas.series.Series,
```

Comment:
where did dataframe and series go? doesn't it need to return the modin dataframe and series now?

Reply:
The type signature here describes which classes use this in their `metaclass` field, e.g. `class DataFrameGroupBy(metaclass=TelemetryMeta)`. Since we no longer declare Series/DataFrame ourselves, we can't apply this metaclass to them.

Instead, we emulate the behavior of the metaclass in `src/.../modin/pandas/__init__.py` by looping over all of Series/DF's attributes and applying the telemetry annotation to each.